nestkernel/source.h

```diff
@@ -40,25 +40,25 @@ namespace nest
 class Source
 {
 private:
-  unsigned long gid_ : 62; //!< gid of source
-  bool processed_ : 1;     //!< whether this target has already been moved
-                           //!to the MPI buffer
+  uint64_t gid_ : 62;  //!< gid of source
+  bool processed_ : 1; //!< whether this target has already been moved
+                       //!to the MPI buffer
   bool primary_ : 1;
-  static const size_t disabled_marker_ = 4611686018427387904 - 1; // 2 ** 62 - 1
+  static constexpr uint64_t GID_DISABLED = 0x3FFFFFFFFFFFFFFF; // 2 ** 62 - 1
```

Comment:
Much better! (And checked)
But maybe even better would be (1 << 62) - 1 ?

Reply:
Yes. This comment has now induced me to refactor the entire class, despite the initial intention to just make it work again on 32-bit. I am going to push a new version.